### PR TITLE
Add giscus

### PR DESCRIPTION
**Pull request by RayzaAnchayhua**
We are waiting the approve for giscus app instalation

### CODE REVIEW

File: blog/templates/layouts/post.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: :theme/post
+---
+
+{@io.quarkiverse.roq.frontmatter.runtime.model.DocumentPage page}
+
+{#insert /}
+
+{#article-end}
+<div class="giscus">aa</div>
```
Comment (mathecruz) [inline blog/templates/layouts/post.html]:
```suggestion
<div class="giscus"></div>
```

File: blog/content/posts/2024-09-20-hightlight-js.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ hljs.highlightAll();
 To style the code blocks, import the Highlight.js default theme into your SCSS file. Add this to your `src/main/resources/web/app/main.scss`:
 
 ```scss
-@import 'highlight.js/scss/default.scss';
+@import "highlight.js/scss/default.scss";
```
Comment (mathecruz) [inline blog/content/posts/2024-09-20-hightlight-js.md]:
```suggestion
@import 'highlight.js/scss/default.scss';
```

File: blog/content/posts/2024-09-20-hightlight-js.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ This will make the Highlight.js library available to your project.
 Now, let’s configure Highlight.js. In your `src/main/resources/web/app/main.js`, import the library and activate it:
 
 ```javascript
-import hljs from 'highlight.js';
+import hljs from "highlight.js";
```
Comment (mathecruz) [inline blog/content/posts/2024-09-20-hightlight-js.md]:
```suggestion
import hljs from 'highlight.js';
```

File: blog/content/posts/2024-09-16-nice-url-experience.md [hunk 4]
```diff
@@ -13,24 +13,21 @@ Managing URLs is now very easy! With our updated Qute-powered feature, you can n
...
-    href="\{post.url.absolute}">
-</a>
+<a class="post-thumbnail" href="\{post.url.absolute}"> </a>
 ```
 
 - ** Smart URL:**
+
 ```html
-<meta name="twitter:image:src" content="\{page.img.absolute}">
+<meta name="twitter:image:src" content="\{page.img.absolute}" />
```
Comment (mathecruz) [inline blog/content/posts/2024-09-16-nice-url-experience.md]:
```suggestion
<meta name="twitter:image:src" content="\{page.img.absolute}" >
```

File: blog/content/posts/2024-09-20-pagination.md [hunk 5]
```diff
@@ -34,15 +32,18 @@ Next, in your template, loop through the paginated posts using:
 To add pagination controls, add something like this to `partials/pagination.html` and include it in your page `\{#include partials/pagination.html/}`:
 
 ```html
-\{#include fm/pagination.html}
-\{#newer}<i class="fa fa-long-arrow-left" aria-hidden="true"></i>\{/newer}
-\{#older}<i class="fa fa-long-arrow-right" aria-hidden="true"></i>\{/older}
-\{/include}
```
Comment (mathecruz) [inline blog/content/posts/2024-09-20-pagination.md]: Could you use the old way?

File: blog/content/posts/2024-10-09-aliases.md [hunk 6]
```diff
@@ -10,9 +10,9 @@ aliases: [aliases-very-cool, aliases-4-ever, aliases-again]
 
 In the last post, we saw how easy it is to use Quarkus for static site generator (@ia3andy's was right!). I am excited to share that we now have a new plugin that allows you to set up redirects for your blog posts! For this post, I've created three aliases.
 
-- [aliases-very-cool]({site.url('/aliases-very-cool')})
-- [aliases-4-ever]({site.url('/aliases-4-ever')})
-- [aliases-again]({site.url('/aliases-again')})
+- [aliases-very-cool](<{site.url('/aliases-very-cool')}>)
+- [aliases-4-ever](<{site.url('/aliases-4-ever')}>)
+- [aliases-again](<{site.url('/aliases-again')}>)
```
Comment (mathecruz) [inline blog/content/posts/2024-10-09-aliases.md:16]: Could you use the old way too.

File: blog/content/posts/2024-09-20-pagination.md [hunk 7]
```diff
@@ -34,15 +32,18 @@ Next, in your template, loop through the paginated posts using:
...
+\{#include fm/pagination.html} \{#newer}<i
+  class="fa fa-long-arrow-left"
+  aria-hidden="true"
+></i
+>\{/newer} \{#older}<i class="fa fa-long-arrow-right" aria-hidden="true"></i
+>\{/older} \{/include}
```
Comment (ia3andy) [inline blog/content/posts/2024-09-20-pagination.md]:
```suggestion
\{#include fm/pagination.html}
\{#newer}<i class="fa fa-long-arrow-left" aria-hidden="true"></i>\{/newer}
\{#older}<i class="fa fa-long-arrow-right" aria-hidden="true"></i>\{/older}
\{/include}
```

**Pull request merged**